bond.py

```diff
@@ -18,6 +18,7 @@ def __init__(self,
         self.oauth_adapter = oauth_adapter
         self.fence_api = fence_api
         self.sam_api = sam_api
+        self.token_store = token_store
```

Comment:
Let's rename `token_store` to `refresh_token_store`.  I'm finding myself confused between `TokenStore` and `FenceTokenStorage`.  I know this was a legacy naming convention, and we don't necessarily need to rename the class here, but since we're introducing the variables, I wouldn't mind at least naming those more clearly here.